### ISSUE TITLE
Function bugs

### ISSUE THREAD

**Issue by Jaylukmann** [Bloceducare/500ngdev4ethTokenReward#3]
I believe the reward mapping ,rateMember and rewardMember functions are not communicating.I tested the code on remix and found out that after rating a whitelisted member 5 times using an admin address which should give 10 points which is equivalent to 1 star and then the rewardmember function should reward the member but this is not the case because balances remain at 0 even after a member is rewarded.May be we need to integrate a proper token code into the rewardToken contract with its total supply,symbol and name which will then be used as a reward .

**Comment by nonseodion** [comment]
Yeah, `reward mapping` wasn't used anywhere in the contract the` rewardMember` function only increases the` balances` mapping but to do that the member must have at least a rating of 3. Balances remained at zero because each member's rating could not be changed this was because memory was used instead of storage as the data location for ` __memberstruct` in `rateMember`. Addresses could not be whitelisted and blacklisted because of this same issue. Check my pull request, I implemented the changes so all the functions work properly now.

**Comment by Jaylukmann** [comment]
@nonseodion that will be great, can't wait to see your PR.

**Comment by Jaylukmann** [comment]
@nonseodion why do you have to merge your PR yourself.We should allow somebody be in charge of merging for easy coordination.If everyone merges their work both good and bad the whole test branch or even master branch will be a mess.

**Comment by nonseodion** [comment]
Really sorry about that I didn't realize that would be the case. Won't happen again.

**Comment by dev-Cnote** [comment]
@Jaylukmann Yeah, I think our mentor will best suited to merge PR's

**Comment by c-n-o-t-e** [comment]
> @nonseodion why do you have to merge your PR yourself.We should allow somebody be in charge of merging for easy coordination.If everyone merges their work both good and bad the whole test branch or even master branch will be a mess.

 Yeah, I think our mentor will best suited to merge PR's

**Comment by bobeu** [comment]
> I believe the reward mapping ,rateMember and rewardMember functions are not communicating.I tested the code on remix and found out that after rating a whitelisted member 5 times using an admin address which should give 10 points which is equivalent to 1 star and then the rewardmember function should reward the member but this is not the case because balances remain at 0 even after a member is rewarded.May be we need to integrate a proper token code into the rewardToken contract with its total supply,symbol and name which will then be used as a reward .

Obviously, Instructor said contract is incomplete as we are yet to mint the token. But it seems that already showed up on the files. Gotta check it out.
